index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+<link rel = "stylesheet" href = "style.css" />
+<title>Portfolio</title>
+</head>
+
+<body>
+    
+<div id = "content">
+   
+    <header>
+        
+        <nav id = "menu">
+            <h4 class = "name">Ivan Gaona</h4>
+            <img src = "images/Union.png" alt = "" height = "25">
+        </nav>    
+
+    </header>
+    
+    <section id="headline-section">
+
+       
+        <h2 class = "title">
+        Ivan Gaona
+        </h2>
+    
+        <h3 class = "subtitle">
+        Software Developer    
+        </h3>
```

Comment:
- [ ] There is a small issue with this text, it doesn't seem to be well aligned (centered) as the one in Figma design see the screenshots below 👇 ,
- Your design
![image](https://user-images.githubusercontent.com/55465133/129815095-706e42a8-2ee1-40f3-9bed-354493eb1085.png)
 
- Figma design
![image](https://user-images.githubusercontent.com/55465133/129815187-3c9c386a-58a0-42c5-8d8f-9156e21908de.png)

Please let us fix this issue 😉